HoloRepositoryUI/client/src/components/app/new_hologram/generate/ImagingStudySelectionStep.tsx

```diff
@@ -1,13 +1,14 @@
 import React, { Component } from "react";
 import { IImagingStudy, IPatient } from "../../../../../../types";
-import { IChoiceGroupOption, IDropdownOption } from "office-ui-fabric-react";
+import { IChoiceGroupOption, IDropdownOption, MessageBar, MessageBarType } from "office-ui-fabric-react";
 import { Col, Divider, Row } from "antd";
 import ImagingStudyDetailsCard from "./ImagingStudyDetailsCard";
 import { PropsWithContext, withAppContext } from "../../../shared/AppState";
 import ImagingStudySelectionInput from "./inputs/ImagingStudySelectionInput";
 import PatientForImagingStudySelectionInput from "./inputs/PatientForImagingStudySelectionInput";
 
-export interface IImagingStudySelectionStepProps extends PropsWithContext {}
+export interface IImagingStudySelectionStepProps extends PropsWithContext {
```

Comment:
Was this change indended? Just wondering if it's a style guide recommendation.

Reply:
No. It's auto-formatted by WebStorm. It used to be synced with the `.prettierrc` config but seems to be out of sync now. I also had to disble the TS auto-formatting temporarily cause it doesn't go togehter with the Python/black pre-commit hooks. Anyway, I'll revert the formatting change.